src/arch/xtensa/boot_loader.c

```diff
@@ -266,8 +295,9 @@ void boot_master_core(void)
 	/* parse manifest and copy modules */
 	platform_trace_point(TRACE_BOOT_LDR_MANIFEST);
 	parse_manifest();
-#endif
 
+	hp_sram_power_off_unused_banks(fw_size_in_use);
```

Comment:
What is the point of powering off banks in `hp_sram_init`, if we are doing it also here?

Reply:
We are ensuring that they are powered on in init by enabling whole memory - hence 

hp_sram_power_off_memory(SOF_MEMORY_SIZE);

change in accordance to @mmaka1 requests